### PR TITLE
Card

### PR DESCRIPTION
**Pull request by ArsallanAbbas**
**Card feature added**

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Header = () => {
+  return (
+    <div>Header</div>
```
Comment (ArsallanAbbas) [inline src/components/Header.jsx:5]: Add another div here

File: src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Header = () => {
+  return (
+    <>
+        <div>Header</div>
+        <div>Another div added</div>
```
Comment (syedzafaryabhaider) [inline src/components/Header.jsx:7]: Add a search bar.

Comment (ArsallanAbbas) [reply src/components/Header.jsx:7]: search bar has been added

Comment (syedzafaryabhaider) [reply src/components/Header.jsx:7]: Ok.

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Header = () => {
```
[the remark this answers is not in the document]
Comment (ArsallanAbbas) [reply src/components/Header.jsx:3]: done

File: src/components/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Card = () => {
+  return (
+    <div>Card</div>
```
Comment (syedzafaryabhaider) [inline src/components/Card.jsx:5]: add loop

**Review by syedzafaryabhaider — APPROVED** [review]

**Pull request merged**